### DISCUSSION TITLE
About translation of dialog buttons

### DISCUSSION THREAD

**Discussion by Prime9999** [dbeaver/dbeaver#21083] [Q&A] [answered]
There seems to be nothing with dialog buttons' text which is not translated in any *.properies file.
How to translate dialog buttons' text?

**Comment by E1izabeth** [comment, marked as answer]
This means that the values are hard-coded in the source code. To be able to translate this into some other language, you need to add localization to these places, following the example of how it is done in other places, or create an issue and we will do it ourselves.
